apps/api/e2e/mock-bridge-server.ts

```diff
@@ -0,0 +1,110 @@
+import http from 'http';
+import express from 'express';
+import { serve, Client } from '@novu/framework/express';
+
+export class MockBridgeServer {
```

Comment:
Not picky here, it just feels like we're creating an actual test server and not a mock.

```suggestion
export class TestBridgeServer {
```

Reply:
Valid. Updated.